content/courses/Guidelines_for_Safe_and_Secure_Ada_SPARK/chapters/guidelines/concurrency/con01_use_the_ravenscar_profile.rst

```diff
@@ -0,0 +1,115 @@
+-----------------------------------
+Use the Ravenscar Profile (CON01)
+-----------------------------------
+
+.. include:: ../../../global.txt
+
+**Level** :math:`\rightarrow` Advisory
+
+Category
+   :Safety: :math:`\checkmark`
+   :Cyber: :math:`\checkmark`
+
+Goal
+   :Maintainability: :math:`\checkmark`
+   :Reliability: :math:`\checkmark`
+   :Portability: :math:`\checkmark`
+   :Performance: :math:`\checkmark`
+   :Security:
+
+**Remediation** :math:`\rightarrow` High
```

Comment:
It is good to explain "Remediation" and its possible values in the Definition page (the same page that "Level" and its possible values are defined) 

It does not look right to me that a guidance level is Advisory but the Remediation is High